contracts/VestingEscrowFactory.vy

```diff
@@ -63,17 +55,23 @@ def deploy_vesting_contract(
     @param vesting_start Epoch time when tokens begin to vest
     """
     assert cliff_length <= vesting_duration  # dev: incorrect vesting cliff
-    escrow: address = create_forwarder_to(self.target)
-    assert ERC20(token).transferFrom(msg.sender, self, amount)  # dev: funding failed
-    assert ERC20(token).approve(escrow, amount)  # dev: approve failed
-    VestingEscrowSimple(escrow).initialize(
+    assert vesting_duration > 0 # dev: duration must be > 0
+    escrow: address = create_from_blueprint(
```

Comment:
let's discuss minimal proxy vs blueprint [trade-offs](https://docs.vyperlang.org/en/stable/built-in-functions.html#chain-interaction). how does the deployment cost change? are the savings on no delegatecall worth it given vesting contracts aren't getting called that often?

Reply:
decided to revert back to minimal proxies based on usage stats of previous versions.